dietpi/dietpi-software

```diff
@@ -10360,10 +10360,12 @@ _EOF_
 			then
 				# Add i386 arch: https://packages.debian.org/bullseye/steam
 				[[ $(dpkg --print-foreign-architectures) == *'i386'* ]] || { G_EXEC dpkg --add-architecture i386; G_AGUP; }
-				# Add Nvidia i386 driver libs if Nvidia driver is detected
-				local nvidia=()
-				[[ -e '/sys/module/nvidia/version' ]] && nvidia=('nvidia-driver-libs:i386')
-				G_AGI steam "${nvidia[@]}"
+
+				G_AGI steam
+				
+				#Reapply GPU drivers to install required i386 items
+				local gpu_current=$(sed -n '/^[[:blank:]]*CONFIG_GPU_DRIVER=/{s/^[^=]*=//p;q}' /boot/dietpi.txt)
+				/boot/dietpi/func/dietpi-set_hardware dedicatedgpu $gpu_current
```

Comment:
Let's fall back to custom/noop in case the setting is not set.
```suggestion
				local gpu_current=$(sed -n '/^[[:blank:]]*CONFIG_GPU_DRIVER=/{s/^[^=]*=//p;q}' /boot/dietpi.txt)
				/boot/dietpi/func/dietpi-set_hardware dedicatedgpu "${gpu_current:-custom}"
```